db/migrate/20190805170949_change_columns_to_not_null.rb

```diff
@@ -0,0 +1,121 @@
+class ChangeColumnsToNotNull < ActiveRecord::Migration[5.1]
+  def change
+    change_column_null :pipeline_run_stages, :pipeline_run_id, false
```

Comment:
For my own education, is this going to cause problems because create_run_stages happens before_create for new pipeline_runs?

https://github.com/chanzuckerberg/idseq-web/blob/3e72558c8538c43fbe7681ba3e9a885e79b62c2b/app/models/pipeline_run.rb#L191